README.md

```diff
@@ -95,16 +97,16 @@ head.link
 head.script
 ```
 
-`head` contains five possible properties, `title`, `base`, `meta`, `link`, `script`:
+`head` contains five possible properties, `htmlAttributes`, `title`, `base`, `meta`, `link`, `script`:
 
-- Each property contains `toComponent()` and `toString()` methods. Use whichever is appropriate for your environment. E.g:
+- Each property contains `toComponent()` and `toString()` methods. Use whichever is appropriate for your environment. For htmlAttributes, use the JSX spread operator on the object returned by `toComponent()`. E.g:
```

Comment:
It's the `ES6 spread operator` right?  want to clarify so someone else doesn't get confused.